common/src/main/java/org/apache/gravitino/utils/TypeValid.java

```diff
@@ -0,0 +1,13 @@
+package org.apache.gravitino.utils;
+
+public class TypeValid {
```

Comment:
Could you rename this to `TypeValidator`?

Reply:
ok , thanks